packages/memcache-client/src/lib/connection.ts

```diff
@@ -342,6 +342,8 @@ export class MemcacheConnection extends MemcacheParser {
   }
 
   _setupConnection(socket: Socket): void {
+    socket.setNoDelay(true);
```

Comment:
Maybe we should make this into an option for the client so that the user can choose if or not they want to use Nagle's algorithm? I think it will come down to if the user wants to prioritize latency or throughput.